### DISCUSSION TITLE
Black artifacts when changing surface alpha

### DISCUSSION THREAD

**Discussion by famrein** [BradyAJohnston/MolecularNodes#329] [Q&A] [answered]
Hi, Im not sure if this is a molecular nodes specific thing. I have a protein that has RNA bound to it on the inside, and im using an empty with geometry proximity to modify the alpha of the surface. This is my node setup:

![image](https://github.com/BradyAJohnston/MolecularNodes/assets/22937598/81a78c6f-0331-40b2-95d5-e8be253be9e7)

And this is what it looks like when i move the empty across the system. I have an array of proteins as if it was a crystal:

https://github.com/BradyAJohnston/MolecularNodes/assets/22937598/51c7206f-d869-4a69-886a-d449c92c5684

**Comment by BradyAJohnston** [comment, marked as answer]
This is not specific to Molecular Nodes, but to do with the rendering engine Cycles. There is a limit to the number of times a light ray will pass through a transparent material. You can increase that amount in the render settings, although a higher count will affect render times: 
![CleanShot 2023-10-19 at 09 27 15@2x](https://github.com/BradyAJohnston/MolecularNodes/assets/36021261/b7e0183b-d4da-4087-88c8-8d42a54af337)


You could also try out transparent materials in EEVEE which don't have a limit (but can be tricky to set up).
